src/controller/Controller.js

```diff
@@ -0,0 +1,86 @@
+import OutputView from '../view/OutputView.js';
+import InputView from '../view/InputView.js';
+import CarModel from '../model/CarModel.js';
+import { validateName, validateTryCount } from '../validation.js';
+import { getRandomNumber } from '../utils/getRandomNumber.js';
+
+export default class Controller {
+  constructor() {
+    this.inputView = new InputView();
+    this.outputView = new OutputView();
+  }
+
+  async start() {
+    const names = await this.getNames();
+    const count = await this.getCount();
+
+    const cars = this.createCars(names);
+    this.outputView.printEmptyLine();
+    this.outputView.printResult();
+
+    this.runRace(cars, count);
+    this.findWinner(cars);
+  }
+
+  runRace(cars, count) {
+    for (let i = 0; i < count; i++) {
+      for (let j = 0; j < cars.length; j++) {
+        const car = cars[j];
+        this.runSingleRound(car);
+      }
+      this.outputView.printEmptyLine();
+    }
+  }
+
+  findWinner(cars) {
+    const max = Math.max(...cars.map((car) => car.getStep()));
+
+    const maxCars = cars.filter((car) => car.getStep() === max);
+    const winners = maxCars.map((car) => car.getName());
+
+    this.outputView.printWinner(winners);
+  }
+
+  runSingleRound(car) {
+    if (this.canMoveForward()) {
+      car.move();
+    }
+
+    this.outputView.printStep(car.getName(), car.getStep());
+  }
+
+  createCars(names) {
+    return names.map((name) => new CarModel(name));
+  }
+
+  canMoveForward() {
+    const randomNumber = getRandomNumber();
+
+    if (randomNumber >= 4) {
+      return true;
+    }
+
+    return false;
+  }
+
+  async getNames() {
+    const rawInput = this.inputView.getInput(
```

Comment:
```suggestion
    const rawInput = await this.inputView.getInput(
```

여기서 await을 사용하면 아래 input을 선언하지 않아도 되지 않을까요?

Reply:
맞아요 저도 남주셔신 리뷰들을 보고 불필요한 변수 선언이었다고 알게됐습니다...😇